be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -189,9 +193,38 @@ class SegmentIterator::BackwardBitmapRangeIterator : public SegmentIterator::Bit
 
         return true;
     }
+    uint32_t read_batch_rowids(rowid_t* buf, uint32_t batch_size) override {
+        if (!_riter.has_value || _rowid_left == 0) {
+            return 0;
+        }
+
+        if (_rowid_count <= batch_size) {
```

Comment:
Is it OK if _rowid_left <= batch_size?

Reply:
No, here we are making a one-time determination of whether the size of the roaring bitmap is smaller than the batch_size, rather than judging the remaining size through '_rowid_left'.